Resources/Prototypes/_CD/Datasets/Names/ipc_last.yml

```diff
@@ -0,0 +1,24 @@
+- type: dataset
```

Comment:
I am not entirely sure that I like the joke of IPC always being named after variables. Could you add some non-variable names?

Reply:
do you have any ideas for them? kinda ran out half way

Reply:
I suck at coming up with names. Parkstation (which is apparently MIT licensed?) has [this list](https://github.com/Simple-Station/Parkstation/blob/4b65cdf873a29cdcc796581a61095ebe54f1c959/Resources/Prototypes/SimpleStation14/Datasets/Names/ipc_names.yml) but that seems kinda boring to me. You could always just google "robot name list" and see if anything interesting comes up.

Reply:
being an IPC gives me the right to make lrp names /j